cli/cmd/cluster/apply.go

```diff
@@ -73,7 +95,7 @@ func Apply(contextLogger *log.Entry, options ApplyOptions) error {
 		ex:            c.terraformExecutor,
 	}
 
-	charts := platform.CommonControlPlaneCharts(options.UpgradeKubelets)
+	charts := removeKubeletChart(c.platform.Meta().ControlplaneCharts, options.UpgradeKubelets)
```

Comment:
Rather than doing this magic, how about we simply use `c.platform.Meta().ControlPlaneCharts` here?

Reply:
There are two streams of control flow coming here. A user may or may not have self-hosted kubelet enabled on the cluster and that piece of information comes from the `lokocfg` file.

And another is whether we want to update kubelet component. We need to remove kubelet from the list if is enabled by the virtue of `disable_self_hosted_kubelet = false` and user does not want to upgrade it.

Reply:
ooh, okay, I missed that part :+1: 

How about we do:
```go
charts := c.platform.Meta().ControlplaneCharts
if !options.UpgradeKubelets {
  charts := removeKubeletChart(charts)
}
```

I think it will be easier to understand then.

Reply:
done.